test/test_helper.rb

```diff
@@ -10,12 +10,15 @@ class ActiveSupport::TestCase
   DatabaseCleaner.strategy = :transaction
 
   def before_setup
+    # FIXME: make sure all users are deleted, sometimes records stick around
+    # when tests are aborted
+    User.with_deleted.map(&:really_destroy!)
```

Comment:
I had trouble with database cleaner before. And it wasn't because of killing the tests. Misteriously multiplying records within `def setup`. E.g. 9 Users after creating one and such.
Thats why I did this "Holzhammermethode" thingie that was here in place before.

I didn't realize it was because of paranoia, but now it makes quite allot of sense. Thanks for the enlightenment.